_locales/_languages.json

```diff
@@ -75,10 +75,6 @@
     "locale": "it",
     "name": "Italian"
   },
-  {
-    "locale": "nl",
-    "name": "Dutch"
-  },
   {
     "locale": "nl_NL",
     "name": "Nederlands (Nederland)"
```

Comment:
although, for whatever reason, "nl_NL" is used as the language code, in the ui, we should not show countries or regions unless we have different language packets for them (true or portuguese). so, the string should just read "Nederlands" (which is an official language eg. also in  Belgium).